app/messages.py

```diff
@@ -75,7 +75,10 @@
 # Update
 NO_DATA_FOR_UPDATING_PROFILE_WAS_SENT = {"message": "No data for updating"
                                          "profile was sent"}
-
+USER_ENTERED_CURRENT_PASSWORD_FOR_NEW_PASSWORD = {"message": "New password cannot be the same "
+                                                  "as the current password"}
+USER_ENTERED_EMPTY_PASSWORD_FOR_NEW_PASSWORD = {"message": "The new_password field has to be longer than 7 characters and "
```

Comment:
Can you check this method `test_user_password_change_request_password_inferior_to_limit` and implement the message like this instead of adding a new one?

Reply:
Done!